src/Storage/appsettings.Development.json

```diff
@@ -1,4 +1,11 @@
 {
+  "PostgreSQLSettings": {
```

Comment:
Looks like the settings reader is case insensitive. The code looks for section `PostgreSqlSettings`.

Reply:
Done